src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixAllCodeAction.cs

```diff
@@ -45,9 +45,15 @@ public override string Title
             }
         }
 
+        internal override string Message => FeaturesResources.ComputingFixAllOccurrences;
+
+        //public override string Title => FeaturesResources.FixAllOccurrences;
+
+        //protected virtual string FixAllWaitDialogAndPreviewChangesTitle => FeaturesResources.FixAllOccurrences;
+        //protected virtual string ComputingFixAllWaitDialogMessage => FeaturesResources.ComputingFixAllOccurrences;
```

Comment:
Did you mean to override `Message` here so that the dialog doesn't show the same thing for both the title and the message?